tmt/steps/provision/artemis.py

```diff
@@ -264,6 +269,8 @@ class GuestArtemis(tmt.GuestSsh):
     api_timeout: int
     api_retries: int
     api_retry_backoff_factor: int
+    watchdog_dispatch_delay: int
+    watchdog_period_delay: int
```

Comment:
These two do not have default values, therefore may remain `None`, `Optional` is therefore needed.

Reply:
Fixed, thanks!